docs/qcqmc/full_workflow.ipynb

```diff
@@ -0,0 +1,363 @@
+    "from recirq.qcqmc.analysis import OverlapAnalysisData, OverlapAnalysisParams\n",
+    "\n",
+    "analysis_params = OverlapAnalysisParams(\n",
+    "    \"TEST_analysis\", experiment_params=experiment.params, k_to_calculate=(1,)\n",
+    ")\n",
+    "all_dependencies: Dict[Params, Data] = {\n",
+    "    pyscf_params: pyscf_hamiltonian,\n",
+    "    pp_params: trial_wf,\n",
+    "    blueprint_params: blueprint,\n",
+    "    simulated_experiment_params: experiment,\n",
+    "}\n",
+    "analysis = OverlapAnalysisData.build_analysis_from_dependencies(\n",
+    "    analysis_params, dependencies=all_dependencies\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally we save the wavefunction in a format that is acceptable for ipie."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.convert_to_ipie import save_wavefunction_for_ipie\n",
+    "ipie_data = save_wavefunction_for_ipie(\n",
+    "    hamiltonian_data=pyscf_hamiltonian, trial_wf_data=trial_wf, overlap_analysis_data=analysis, do_print=False\n",
+    ")\n",
+    "print(f\"Reconstructed shadow wavefunction energy: {ipie_data.variational_energy}\")\n",
+    "print(f\"Ideal trial wavefunction energy: {trial_wf.ansatz_energy}\")\n",
+    "print(f\"FCI energy: {trial_wf.fci_energy}\")\n",
+    "print(ipie_data)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run AFQMC \n",
+    "\n",
+    "Now that we have built our quantum wavefunction we can use it as a trial wavefunction in an AFQMC simulation. First we need to build a factorized Hamiltonian which is required for AFQMC. \n",
+    "\n",
+    "In particular we require the three-index Cholesky tensor (`ham.chol` below) which satisfies\n",
+    "\n",
+    "$$\n",
+    "(pq|rs) = \\sum_X L_{pq}^X L_{rs}^X.\n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipie.systems.generic import Generic\n",
+    "from ipie.utils.from_pyscf import generate_hamiltonian_from_chk\n",
+    "\n",
+    "num_elec = (pyscf_params.n_elec // 2,) * 2\n",
+    "system = Generic(num_elec)\n",
+    "chk_path = pyscf_params.base_path.with_suffix(\".chk\")\n",
+    "ham = generate_hamiltonian_from_chk(str(chk_path))\n",
+    "assert ham.H1.shape == (2, pyscf_params.n_orb, pyscf_params.n_orb)\n",
+    "assert ham.chol.shape[0] == pyscf_params.n_orb * pyscf_params.n_orb"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we need to build a trial wavefunction from quantum wavefunction. In practice the quantum trial is expanded as a linear combination of (orthogonal) Slater Determinants. Within ipie this style of trial wavefunction is defined as a `ParticleHole` trial wavefunction. "
```

Comment:
Comma after "Next",  "In practice" , and also "Within ipie"

Reply:
Done, done, done.